tasks/config.yml

```diff
@@ -10,7 +10,7 @@
 - name: Create an admin user.
   become: true
   become_user: "{{ superset_user }}"
-  shell: "export PYTHONPATH='{{ superset_python_path }}' && {{ superset_virtualenv_path }}/bin/fabmanager create-admin --app {{ superset_app }} --username {{ superset_username }} --firstname {{ superset_firstname }}  --lastname {{ superset_lastname }}  --email {{ superset_email }} --password {{ superset_pass }}"
+  shell: "export PYTHONPATH='{{ superset_python_path }}' && export FLASK_APP='{{ superset_app }}' && {{ superset_virtualenv_path }}/bin/flask fab create-admin --username {{ superset_username }} --firstname {{ superset_firstname }}  --lastname {{ superset_lastname }}  --email {{ superset_email }} --password {{ superset_pass }}"
```

Comment:
@nateMJ assuming this command is not available in older versions of superset, let's say 0.29.0; do you think 1. we should add backwards compatibility for installing these versions and 2. that a good way of allowing backwards compatibility would be to put the commands that have changed into variables, and setting the default values to those supported in 0.34.0?

Reply:
Yes, they are not compatible with the older superset versions. I agree with approach 2. I will change them to variables